golangci-lint | sloppyLen error from gocritic

From a recent linting test failure:

```
main.go:141:6: sloppyLen: len(fileMatches) <= 0 can be len(fileMatches) == 0 (gocritic)
```

I've not looked into it yet, but i suspect I was checking for `<=0` because I was trying to guard against a potentially negative `int` value.

--- a/main.go
+++ b/main.go
@@ -138,7 +138,7 @@ func main() {
 		log.Debugf("Length of matches slice: %d", len(fileMatches))
 
 		log.Debugf("Early exit if no matching files were found.")
-		if len(fileMatches) <= 0 {
+		if len(fileMatches) == 0 {
 
 			log.WithFields(logrus.Fields{
 				"path":         path,

